app/models/compute_resource.rb

```diff
@@ -132,7 +132,7 @@ def host_compute_attrs(host)
 
   def host_interfaces_attrs(host)
     host.interfaces.select(&:physical?).each.with_index.reduce({}) do |hash, (nic, index)|
-      hash.merge(index.to_s => nic.compute_attributes)
+      hash.merge(index.to_s => nic)
```

Comment:
> hash.merge(index.to_s => { :ip => nic[:ip], **(nic[:ip6] ? {:ip6 => nic[:ip6]} : {}) })

This will overwrite the first `index.to_s` entry that had `nic.compute_attributes`, so use:

    hash.merge(index.to_s => nic.compute_attributes.merge(ip: nic.ip, ip6: nic.ip6))

Don't worry about making ip6 optional, the patch should only be written for develop, not for backwards compatibility.